Osiris/SDK/WeaponId.h

```diff
@@ -178,5 +188,12 @@ constexpr int getWeaponClass(WeaponId weaponId) noexcept
     case WeaponId::Awp:
     case WeaponId::G3SG1:
     case WeaponId::Scar20: return 38;
+
+    case WeaponId::Flash:
+    case WeaponId::Decoy:
+    case WeaponId::Grenade:
+    case WeaponId::IncGrenade:
+    case WeaponId::Molotov:
+    case WeaponId::Smoke: return 40;
```

Comment:
This will make aimbot / triggerbot crash. `getWeaponClass()` function is only meant to be used in aimbot & triggerbot.